tests/tests_plotting/test_misc.py

```diff
@@ -5,11 +5,19 @@
 from poliastro.plotting.misc import plot_solar_system
 
 
+@pytest.mark.filterwarnings(
```

Comment:
Instead, let's use a date in the test that doesn't yield this warning. Is this possible?

Reply:
@astrojuanlu these warnings can't be removed as they originate upstream in astropy using erfa module where https://pyerfa.readthedocs.io/en/latest/api/erfa.epv00.html can even use negative value of dates.

Reply:
I suspect that `plot_solar_system` is causing this. Some planets might have a period larger than 90 years, therefore more time is needed to compute a full revolution.